examples/simple_calculator/src/aiq_simple_calculator/configs/config.yml

```diff
@@ -15,6 +15,13 @@
 
 general:
   use_uvloop: true
+  front_end:
```

Comment:
Instead of adding this to the base config.yml can you please create a new one , config_metadata.yml?

Add a section to the README to explain its usage.